data/json/monsters.json

```diff
@@ -2510,6 +2549,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_predator",
+    "name": "feral predator",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"brown_green",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":6,
+    "aggression":100,
+    "morale":100,
+    "speed":170,
```

Comment:
Ah, that's what that value does. And I meant for it to be very quick, but it has a fairly low heath value to compensate.